labrad/server.py

```diff
@@ -352,6 +363,17 @@ def startup(self, protocol):
             self.disconnect(e)
             raise
 
+    @property
+    def client(self):
+        if threadable.isInIOThread():
+            # We're in the reactor thread, so can return shared async client.
+            return self.__async_client
+        else:
+            if not hasattr(self.__sync_client, 'cxn'):
+                backend = labrad.backend.TwistedConnection(async_cxn=self.__protocol)
+                self.__sync_client.cxn = Client(backend)
+            return self.__sync_client.cxn
```

Comment:
What's unclear to me here is the return type -- is `self.__sync_client.cxn` an `AsyncClient` object (or something compatible)?